server/src/HTTPMethodInvalid.hs

```diff
@@ -0,0 +1,28 @@
+module HTTPMethodInvalid (canonicalizeHTTPMethods, limitHTTPMethods) where
+
+import qualified Network.Wai as Wai
+import qualified Network.HTTP.Types as HTTP
+import qualified Network.HTTP.Types.Method as Method
+import qualified Data.ByteString.Char8 as ByteString
+import qualified Data.ByteString.Lazy.Char8 as LBS
+
+-- Checks that the HTTP method is one of the StdMethods.
```

Comment:
You can use Haddock comments for these functions; https://www.haskell.org/haddock/doc/html/markup.html